src/NorthwindWebsite/Presentation/HtmlHelpers/NorthwindImageTagHelper.cs

```diff
@@ -0,0 +1,27 @@
+﻿using Microsoft.AspNetCore.Razor.TagHelpers;
+
+namespace NorthwindWebsite.Presentation.HtmlHelpers;
+
+[HtmlTargetElement("northwind-image", TagStructure = TagStructure.NormalOrSelfClosing)]
+public class NorthwindImageTagHelper : TagHelper
+{
+    private const string ImageUploadLinkPatern = "/Categories/ImageUpload/";
```

Comment:
1. Pattern*
2. I think `Pattern` postfix is quite redundant, you can just remove it and sense will not be changed